modin/pandas/test/test_rolling.py

```diff
@@ -34,7 +35,19 @@
 # have too many such instances.
 # TODO(https://github.com/modin-project/modin/issues/3655): catch all instances
 # of defaulting to pandas.
-pytestmark = pytest.mark.filterwarnings(default_to_pandas_ignore_string)
+pytestmark = [
+    pytest.mark.filterwarnings(default_to_pandas_ignore_string),
+    # TO MAKE SURE ALL FUTUREWARNINGS ARE CONSIDERED
+    pytest.mark.filterwarnings("error::FutureWarning"),
+    # IGNORE FUTUREWARNINGS MARKS TO CLEANUP OUTPUT
+    pytest.mark.filterwarnings(
+        "ignore:Support for axis=1 in DataFrame.rolling is deprecated:FutureWarning"
+    ),
+    # FIXME: these cases inconsistent between modin and pandas
```

Comment:
in which way they're inconsistent, do we have an issue for that?

Reply:
Modin does not show this warning in cases where Pandas does. Not yet.

Reply:
@dchigarev let's merge, I will create issue later.